rust/gui-client/src-tauri/src/client/gui/system_tray.rs

```diff
@@ -510,48 +484,38 @@ mod tests {
                     .copyable("test")
                     .copyable(GATEWAY_CONNECTED),
             )
+            .add_submenu(
+                "Internet Resource",
+                Menu::default()
+                    .separator()
```

Comment:
I think there might be an extra separator at the top and bottom.

```suggestion
```

Reply:
I just kept the same separators from the normal resource, in that case the first one is before "Resource" which do look weird for the internet resource I can remove. But the one below is before the "Site" section